src/oic/oic/provider.py

```diff
@@ -2191,6 +2197,10 @@ def end_session_endpoint(
                     redirect_uri = "%s?%s" % (_base, query_string)
                 else:
                     redirect_uri = _base
+            else:
+                raise ValueError(
+                    "Missing post_logout_redirect_uri and more then one post_logout_redirect_uris"
```

Comment:
Shouldn't this result in `error_response` back to the client?

Reply:
Probably